### ISSUE TITLE
Android Notifications don't work

### ISSUE THREAD

**Issue by CronobotTech** [ConnectyCube/android-messenger-app#79]
Hi,
**TL;DR:** I'm a learning developer and I need help with setting up notifications. I've tried doing exactly as shown in the guide many times but I just don't seem to get notifications at all. If someone could please explain what I need to do and where I would really appreciate it.

What I've done so far:
- Configured Firebase project and Server key
- Prepared app dependencies
- Prepared app AndroidManifest but 
`<service
   android:name="com.connectycube.pushnotifications.services.fcm.FcmPushInstanceIDService">
   <intent-filter>
       <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
   </intent-filter>
</service>` 
didn't work because FcmPushInstanceIDService wasn't available. I'm guessing I don't need this but If I do, please let me know.
- Tried to Enable/Disable push subscription but didn't know where to implement the given piece of code though I did try to Implement it in SettingsProvider.kt in an init function.

**Note: After doing all of this, I checked the ConnectyCube admin console and in the Device Subscriptions panel, I could see that two of my Users were subscribed but it showed channel "GCM", and I'm guessing it was supposed to be FCM. Also, in the Queue, I saw a few notifications with different IDs (if that matters) but once again, the channel was GCM.**

I've done tried all of the above but it doesn't seem to work, If I'm getting something wrong here, please let me know. And any help will be appreciated.

Thank You.

**Comment by TatankaConCube** [comment]
hello @CronobotTech, we saved 'GCM' name just for supporting old versions of our SDK, really it is FCM push notifications. 
We already implemented push notifications in our [android-messenger-app](https://github.com/ConnectyCube/android-messenger-app), just review its code and search for your mistakes. Pay attention to point 4 of [our guide](https://github.com/ConnectyCube/android-messenger-app#how-to-run) for running this sample app.

**Comment by TatankaConCube** [comment]
No activity for a long time. Closing...
